documentation/src/main/markdown/properties/configuration/component-location-analysis.md

```diff
@@ -0,0 +1,15 @@
+# component-location-analysis
+
+Enable this feature by adding --component.location.analysis.enabled=TRUE to a run of Detect.
+
+When enabled, Synopsys Detect creates an output file in the scan subdirectory of the output directory with the name 'components-with-locations.json' which identifies the declaration locations (filepath, line number and column position) of open source components found in the source project.
+
+* As Detect shuts down, by default, it performs cleanup operations which include deleting the component location analysis file. You can disable clean up by setting --detect.cleanup=false.
+
+## Requirements and Limitations
+
+* A limited subset of Tools can be run.
+    * The currently supported tools are: DETECTOR.
```

Comment:
All other tools are disabled when running in Rapid Scan mode? Should we mention it here?